src/edit/InstanceForm.js

```diff
@@ -199,6 +194,19 @@ class InstanceForm extends React.Component {
     );
   }
 
+  getActionMenu() {
+    const { onCancel } = this.props;
+    return (
+      <Fragment>
+        <Button buttonStyle="dropdownItem" id="cancel-instance-edition" onClick={onCancel}>
```

Comment:
I know there are no conventions about props quantity inside tag element. In my opinion when we have more than one prop it should be placed on separated lines to increase code readability. Does it make sense? What do you think?

Reply:
I think that's a great convention 👍

Reply:
I guess my thought was that more actions would be added in the future and having the Fragment would show that it's recommended to use a Fragment for this purpose oppose to a div. But of course, that would probably be a given for any developer extending this functionality in the future so no need to have it there for now.

I'll remove the Fragments! 👍